@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardHostsPatches.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <UiCard>
+    <UiCardTitle class="patches-title">
+      {{ $t("patches") }}
+      <template v-if="areAllLoaded" #right>
+        {{ $t("missing", { n: count }) }}
+      </template>
```

Comment:
My first version had a spinner, but it looked weird since each row already has one while not all hosts patches are loaded.

![Patches loading](https://user-images.githubusercontent.com/19408/227149973-c0bf6428-c93b-4f99-86d3-936f5f5c7b59.png)